test/convert_test.go

```diff
@@ -63,13 +64,19 @@ func TestConvert(t *testing.T) {
 			t.Fatalf("unmarshaling: %v", err)
 		}
 
-		assetsByType := make(map[string][]google.Asset)
+		assetsByType := make(map[string]google.Asset)
 		for _, a := range assets {
-			assetsByType[a.Type] = append(assetsByType[a.Type], a)
+			t := a.Type
```

Comment:
Let's not override t.

```suggestion
			aType := a.Type
```

Reply:
Oh good call haha...totally forgot that's a special var.